specification/azsadmin/resource-manager/compute/Microsoft.Compute.Admin/preview/2018-07-30-preview/examples/Disks/List.json

```diff
@@ -21,7 +21,8 @@
               "status": "Unattached",
               "sharePath": "\\\\SU1FileServer.mydomain.com\\SU1_ObjStore_4",
               "actualSize": 183,
-              "tenantId": "/subscriptions/4496690e-348e-4dc0-addc-17395c3966ba/resourceGroups/DISKRG6ECD14E6/providers/Microsoft.Compute/Disks/diskrptest9vb55sb15w",
+              "provisionSize": 3221225472,
```

Comment:
What are the units?

Reply:
I will update it to be provisionSizeGB. currently it is in GB.

Reply:
I will update it to be provisionSizeGB. Currently it is in bytes. thanks.